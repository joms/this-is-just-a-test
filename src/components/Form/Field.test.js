import React from 'react';
import { shallow } from 'enzyme';
import { Field } from '.';

describe('<Field>', () => {
    it('matches snapshot', () => {
        const wrapper = shallow(<Field>Hello, world!</Field>);
        expect(wrapper).toMatchSnapshot();
    });
});
