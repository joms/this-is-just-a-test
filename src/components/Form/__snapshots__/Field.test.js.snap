// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Field> matches snapshot 1`] = `
<div
  className="field"
>
  Hello, world!
</div>
`;
